$NetBSD$

Configure will record that we are unprivileged during packaging and wire
fix-map accordingly.  However, on pkgsrc, fix-map runs later and should
have no issues with privileged operations (like chown).

--- homedir/fix-map.in.orig	2017-02-04 19:18:32.000000000 +0000
+++ homedir/fix-map.in
@@ -42,7 +42,6 @@
 set -e
 
 @NO_SUID@
-@PKG_MAKE@
 
 DCC_HOMEDIR=@prefix@
 CDCC=@bindir@/cdcc
